doc/guides/sample_app_ug/cndpfwd.rst

```diff
@@ -201,6 +201,7 @@ The configuration json file is located in the ``cndpfwd`` example sub-directory
         //   mode       - (O) Mode type [drop | rx-only], tx-only, [lb | loopback], fwd, tx-only-rx
         //                    acl-strict, acl-permissive
         //   uds_path   - (0) Path to unix domain socket to get xsk map fd
+        //   xsk-pin-path - (O) Path to pinned bpf map
```

Comment:
Should we highlight that this is used in combination with the unprivileged flag?
I guess I could change the implementation to not rely on it... Or maybe a better approach it to have the inhibit prog load flag throughout the apps rather than "unprivileged" as the intention is probably better described through the `inhibit_prog_load`